### DISCUSSION TITLE
백엔드에 인가코드 넘겨준 후 JWT수신 에러

### DISCUSSION THREAD

**Discussion by gygy7151** [EasyAndBeauty/DailyReceipt-FE-WEB#211] [Q&A]
<img width="1182" alt="image" src="https://user-images.githubusercontent.com/55748886/208299655-4321d15e-75b7-4cf9-8be6-d07886e593c7.png">

[ 문제 발견 시나리오 ]
- 도커로 서버 띄우기 성공
- 카카오 로그인 시도

[문제현상]
- AuthPage에 accessToken을 넘겨주는 파라미터 res가 undefined값으로 받아짐 

[확인내용]
- [x] 인가코드는 올바르게 URL에 전송됨을 네트워크 탭을 통해 확인

선배님들의 조언 부탁드립니당🙇🏻‍♀️

**Comment by mdgarden** [comment]
- 백엔드 코드는 최신상태인가요?
- 도커는 데이터베이스의 역할을 합니다. 백엔드는 인텔리제이에서 돌아가는 스프링부트입니다. 도커와 함께 인텔리제이에서 스프링부트가 정상적으로 돌아가고 있는지 확인해주세요.
- 네트워크 오류를 확인할 때는 네트워크 오류코드도 함께 보여주세요. 단순히 오류가 나타났다는 증상만으로는 원인을 파악하기가 어렵습니다.

**Reply by gygy7151** [reply]
- 네트워크 오류 코드 스샷 첨부드립니다 마당~ 🙇🏻‍♀️
 <img width="1184" alt="image" src="https://user-images.githubusercontent.com/55748886/208302282-dbabaee1-e45e-417a-b05d-b74201d90e3f.png">


- 백엔드 코드가 최신 상태가 아님을 확인했습니다.
 main브랜치에서 git fetch origin main -> git merge FETCH_HEAD 한후 병합충돌을 해결하고 다시 인텔리 제이를 돌렸는데, 아래와 같이 2개의 테이블이 없다는 에러가 나옵니다 마당
해당파일은 어느 브랜치에서 pull해야 하는지 알려주실 수 있을까여? @mdgarden 
<img width="1414" alt="image" src="https://user-images.githubusercontent.com/55748886/208302827-be449724-d18c-453f-9bf4-96c060734dab.png">

**Reply by mdgarden** [reply]
네트워크 오류 보여주실 때는 스테이터스 코드와 리스폰스도 함께 보여주세요 ㅠㅠ(400번대인지, 500번대인지...)
도커에 있는 DB삭제하시고 다시 컴포즈 업 하시면 될 것 같습니다.

**Reply by gygy7151** [reply]
마당 저두 그 status code가 너무 궁금한데요 아무리 API요청을 보내도 스샷내용처럼 General과 RequestHeader만 나오더라구여
요청에 실패했을때 400번대인지 500번대인지 그리고 ResponseHeader도 같이 나와야되는데 안보였어욥 ㅠㅠ
애초에 ResponseHeader가 없는 HTTP요청도 존재하나여? 아이러니해용🤔

도커에 있는 DB 삭제하고 다시 컴포즈업 해보겠습니당!!
늦은시간까지 답변 고맙습니다 마댜아아아아아앙🙇🏻‍♀️💚
